docs/DeveloperGuide.md

```diff
@@ -594,7 +592,7 @@ Use case ends.
 
 * 3a. The contact does not have github information.
     
-   Use case resumes from step 2.
+   Use case resumes at step 2.
 
 **Use case: UC13 Undo recent delete/clear operation**
```

Comment:
@huanghao1998 , should we consider removing steps 1 and 2, as they conflict with Extension 3a? I added a similar UC for redo with that in mind, however, if you feel like it is necessary I will change UC14 to follow the format for UC13

Reply:
Yes I agreed!! We should remove step 1 and 2 and follows the UC14